Remove example models from docs.open223.info (#26)
open223/docs.open223.info: reference/open223-resources.md

## [Open223 Models](https://models.open223.info/)

This site contains the canonical sources for example models.
Each page listed in the table of contents (on the left-hand side of the page) describes an example building or system modeled in 223.
On each page you will find the following:
- short description of what is being modeled
- supplementary materials and/or diagrams, where applicable
- download links for the model (both Turtle and JSON-LD are provided)

> - links to example queries against the model; queries are hosted on the [Open223 query page](https://query.open223.info)

Reviewer: ```suggestion
- links to example queries against the model; queries are hosted on Open223 Query (see link below)
```

Or replace with link to the Open223 Query heading below?

- a table of entities found inside the model

## [Open223 Explore](https://explore.open223.info/)

This page hosts autogenerated documentation of the 223 ontology, emphasizing the availability and correct use of concepts and properties defined by the ontology.
The Explore tool divides the ontology up across four tabs.

![](/_static/images/explore-tabs.png)

> #### Classes

Reviewer: ```suggestion
### Classes
```

The classes tab organizes the classes of the standard based on their class hierarchy.

> One can click on the class name to view a description of the class and its constraints, or click on the "+" to the left of the class name to view all the subclasses of that class.

Reviewer: ```suggestion
One can click on the class name to view a description of the class and its constraints, or click on the ➕ to the left of the class name to view all the subclasses of that class.
```

Reviewer: or `+`

Author: I'd rather not have this be a unicode character to aid with screenreaders, etc

> For example, all 223 classes are a subclass of `s223:Concept`, and can be accessed by clicking the "+" to the left of "Concept".

Reviewer: ```suggestion
For example, all 223 classes are a subclass of `s223:Concept`, and can be accessed by clicking the ➕ to the left of "Concept".
```

Reviewer: or `+`

![](/_static/images/explore-classes-domainspace.png)

The **Definition** section describes what relations *instances* of the class must have.
The **Properties** section below lists all of the relation constraints the class as inherited, including: which of those relations are optional (marked with a `?`), can happen 0 or more times (marked with a `*`), must happen

> at least once (marked with a `+`), or are necessary (no additional markings, like in the screenshot above).

Reviewer: ```suggestion
at least once (marked with a `+`), or are necessary (no additional markings, like in the image above).
```

Additionally all links highlighted in yellow on a given class page can be used to navigate to a description of that class or property.

> The definitions listed on the Explore tool are *not* definitive but instead capture common usage; refer to the standard for the full and complete definition.

Reviewer: We *might* include a copy of the Standard's Definitions section. Update later if so.

> #### Properties

Reviewer: ```suggestion
### Properties
```

The properties tab organizes the *relations* of the standard and its dependencies.
Clicking on a relation shows which classe are in the domain and range of the relation, if available.

![](/_static/images/explore-properties-connectedTo.png)

> #### Ontologies and Files

Reviewer: ```suggestion
### Ontologies and Files
```

These last two tabs list *all* properties, concepts, classes, etc defined in each ontology loaded into the tool.
This currently includes 223 as well as its dependencies (e.g., QUDT).

## [Open223 Query](https://query.open223.info/)

This page provides client-side SPARQL query processing using Oxigraph. It contains several example 223 models and several example queries.
There are three dropdowns in the top-left corner of the page:

![](/_static/images/query-open223-loading.png)

Two of these determine which 223 model you want to query:

- You can **upload your own graph** by choosing a local Turtle file to upload or by referencing one by URL on the web

> - You can select an **example 223P model** from the dropdown menu. Each of these models contains a copy of 223 and has had SHACL inference applied to it

Reviewer: ```suggestion
- You can select an **example 223 model** from the dropdown menu. Each of these models contains a copy of 223 and has had SHACL inference applied to it
```

The last dropdown contains a set of example queries against 223 models. Select a query from the dropdown to populate the query box.
You can also edit the query box directly to express a SPARQL query.

> Click the black triangle in the query window to execute the query against the currently loaded model.

Reviewer: ```suggestion
Click the ▶️ in the query window to execute the query against the currently loaded model.
```
